flink-runtime/src/test/java/org/apache/flink/runtime/io/network/NettyShuffleEnvironmentTest.java

```diff
@@ -130,18 +130,18 @@ private void testRegisterTaskWithLimitedBuffers(int bufferPoolSize) throws Excep
 		final SingleInputGate[] inputGates = new SingleInputGate[] {ig1, ig2, ig3, ig4};
 
 		createRemoteInputChannel(ig4, 0, rp1, connManager, network.getNetworkBufferPool());
-		createRemoteInputChannel(ig4, 0, rp2, connManager, network.getNetworkBufferPool());
```

Comment:
I guess this channel index change is irrelevant?

Reply:
yes, it is irrelevant.